src/main/resources/public/js/pages/organization-search.js

```diff
@@ -104,10 +104,9 @@ class OrganizationSearch extends React.Component {
                         }}
                         placeholder={i18n._(t`search.organization.search-organization`)}
                     />
+                    {i18n._(t`organization.search.create`)}
                     <div className="flex-row options">
-                        <Link to="/my/organization/create" className={"new-organization"}>
```

Comment:
the <Link/> component is important and must not be removed (see https://reacttraining.com/react-router/web/api/Link).